### ISSUE TITLE
WCAG 1.4.4 🟠 (Partially Supports): Resize Text

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#5117]
## WCAG 1.4.4: Resize Text

**Level:** AA
**Principle:** Perceivable
**Result:** 🟠 Partially Supports

### Resources
- [WCAG 2.2 Success Criterion 1.4.4](https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html)
> Except for [captions](https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html#dfn-captions) and [images of text](https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html#dfn-image-of-text), [text](https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html#dfn-text) can be resized without [assistive technology](https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html#dfn-assistive-technology) up to 200 percent without loss of content or functionality.

### Audit Checklist
- [x] Review previous Audit Result
- [x] Review work identified in previous audit and list open issues.
- [x] Check for any other relevant open issues.
- [x] Retest open issues, are these issues still present?
- [x] Test for any new issues that may have been introduced.
- [x] Determine compliance status, add this to issue name.
- [x] Add any new issues to the Accessibility project for triage.

### Previous audit
#4127 
> ❌ resize text
Text resizing via page zoom is handled dynamically alongside screen width changes. This works well apart from a known issue with some header images which flow over the text.

 ### Progress since last audit

- #3942
- #4034
- #755
- #4052
- #4684

### Automated Testing
All pages tested passed the axe rules 'meta-viewport' and 'meta-viewport-large' which test that zooming and scalling is not disabled and that text can be scaled up to 500% respectively.  

 ### Audit Notes
While we support the ability to resize text, the results from testing text reflowing showed that there are places where the space around text does not resize as the text size increases, such that part of the text is cut off and content is lost. So we cannot claim to fully support resizing text either. 

 ### Work arising

- #5135
